apps/site-app/src/containers/App/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -1,25 +1,87 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
```

Comment:
I suspect we will remove this test fairly quickly, I'm not at all sure it's a good idea to do snapshot testing (yet)